.travis.yml

```diff
@@ -58,6 +60,10 @@ install:
   - sudo su $GALAXY_TRAVIS_USER -c 'sed -i.bak "s/commands.*$/commands =/" tox.ini'
 
 script:
+  # Test submitting jobs to an external slurm cluster
+  - docker tag galaxy-docker/test quay.io/bgruening/galaxy
+  - cd $TRAVIS_BUILD_DIR/docs/slurm/test && sudo -E su $GALAXY_TRAVIS_USER -c 'bash test.sh'
+  - sudo chown -R $GALAXY_TRAVIS_USER:$GALAXY_TRAVIS_USER /tmp
```

Comment:
Yes, and in addition this needs to be a location that the galaxy user in the container has write access to. I have given it a complete new attempt in 1072a72, which removes all the sudo user manipulation, and instead I'm spinning up the container with the travis' user ID.